Bugfix in fixImage.sh

AndrewFasano opened the pull request:
Simple bugfix, happy to explain more if necessary. Fixes the issue I described in https://github.com/firmadyne/firmadyne/pull/162#issuecomment-720771869

ddcc commented:
Thanks for point this out. Shouldn't the correct fix be to prepend `$BUSYBOX ` to all of the commands; e.g. cp, mv, rm, mknod, and mkdir?

brianpow commented:
Would you check if your host busybox has 'mv' built-in ?

ddcc commented:
All of those commands are included in busybox for me, including mv.

AndrewFasano commented:
Just updated this to use `$BUSYBOX mv` instead. Also changed the other commands to be prefixed with `$BUSYBOX` which is probably safer even though they all seemed to be working without it.

0BuRner commented on scripts/fixImage.sh:
```diff
@@ -15,50 +15,50 @@ resolve_link() {
 if [ ! -s "$PASSWD" ]; then
     echo "Creating $PASSWD!"
-    mkdir -p "$(dirname $PASSWD)"
+    $BUSYBOX $BUSYBOX mkdir -p "$(dirname $PASSWD)"
```
Mistake?
